test/cert-ceremonies/generate.go

```diff
@@ -73,16 +73,18 @@ func genCert(path string) error {
 }
 
 func main() {
+	_ = cmd.NewLogger(cmd.SyslogConfig{StdoutLevel: 6, SyslogLevel: -1})
```

Comment:
Although this definitely works, it also does a bunch of other stuff (e.g. initializing log handlers for mysql, grpc, etc) that's not necessary. I think it would be simpler, and have less magic side-effects, to do something like:
```suggestion
	blog.Set(blog.StdoutLogger(6))
```